docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,43 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
 
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | A method where each voter can select up to one candidate.                        |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | `Approval voting`_, where each voter can select as many candidates as desired.   |
```

Comment:
I can't imagine caring about the distinction in question here enough to think about it for more than three seconds.